tests/integration/api_container_test.py

```diff
@@ -620,6 +620,29 @@ def test_create_with_volume_mount(self):
         assert mount['Source'] == mount_data['Name']
         assert mount_data['RW'] is True
 
+    @requires_api_version('1.45')
+    def test_create_with_subpath_volume_mount(self):
+        mount = docker.types.Mount(
+            type="volume", source=helpers.random_name(),
+            target=self.mount_dest, read_only=True,
+            subpath='subdir'
```

Comment:
From a quick glance, it doesn't seem that the subpath is being checked in this test

Reply:
Made changes accordingly